hack/ci/e2e.sh

```diff
@@ -64,11 +64,20 @@ build() {
         /usr/local/bin/create_bazel_cache_rcs.sh || true
     fi
 
-    # build the node image w/ kubernetes
-    # TODO(bentheelder): remove the kube-root flag after we make kind try to
-    # find this in a go module compatible way
-    kind build node-image --type=bazel \
-        --kube-root="$(go env GOPATH)/src/k8s.io/kubernetes"
+    ARCH=$(dpkg --print-architecture)
```

Comment:
Should we use go to detect the architecture? Dpkg sounds very specific

Reply:
Sure, I just copied this from the build base image docker file, I can change it